### PR TITLE
css fix for button position.

### PR DESCRIPTION
**Pull request by dmpl1**
### Requirements

* sls-app issue fix

### Description of the Change

Test function button position fixed at service and environmental level

### CODE REVIEW

File: core/jazz_ui/src/app/pages/service-detail/service-detail.component.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1 class="page-hdr bold-title">{{service.name}}</h1>
                 <span *ngIf="service.runtime"> - {{service.runtime}}</span>
             </span>
             </div>
-            <div class="image-btn-div">
+            <div class="image-btn-div flex-end">
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-detail/service-detail.component.html]: Why is this changed needed in the service detail? What issue is this addressing?

Comment (dmpl1) [reply core/jazz_ui/src/app/pages/service-detail/service-detail.component.html]: CSS class was added in service-detail as a precaution and to have code in sync between env-detail and service-detail.
